plugins/postcss-gradients-interpolation-method/.tape.mjs

```diff
@@ -36,6 +29,16 @@ postcssTape(plugin)({
 			preserve: false
 		}
 	},
+	'variables:preserve-true': {
+		message: 'supports { preserve: true, enableProgressiveCustomProperties: true } usage',
+	},
+	'variables:preserve-true:progressive-false': {
+		message: 'supports { preserve: true, enableProgressiveCustomProperties: false } usage',
+		options: {
+			preserve: true,
+			enableProgressiveCustomProperties: false,
+		}
+	},
```

Comment:
These tests are intended for variables, so using the variables source file makes more sense.